setup.py

```diff
@@ -47,9 +67,6 @@
         "Operating System :: Unix",
         "Operating System :: MacOS",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
```

Comment:
Are you comfortable with removing support for those versions of Python? In fact, it is only about the software tests, which utilize `importlib.resources.files`, only available on Python>=3.9.

Bringing in a backward-compatibility package, in this case [importlib-resources](https://pypi.org/project/importlib-resources/), would not be that hard, and just a few more lines of code away.

So, just let me know if you would prefer retaining support for those.

Reply:
a333ab6e3351 re-establishes compatibility of the test suite with Python 3.6. The main code base was compatible with Python 3.6 before, and did not undergo any changes in this regard. So, that patch is exclusively about adjusting the test suite.